gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -755,7 +756,7 @@ export class AdvancedDynamicTexture extends DynamicTexture {
             this._shouldBlockPointer = false;
             // Do picking modifies _shouldBlockPointer
             let pointerId = (pi.event as PointerEvent).pointerId || this._defaultMousePointerId;
-            this._doPicking(x, y, pi, pi.type, pointerId, pi.event.button, (<MouseWheelEvent>pi.event).deltaX, (<MouseWheelEvent>pi.event).deltaY);
+            this._doPicking(x, y, pi, pi.type, pointerId, pi.event.button, (pi.event as unknown as IWheelEvent).deltaX, (pi.event as unknown as IWheelEvent).deltaY);
```

Comment:
why as unknown?

Reply:
Originally, that was a carry over from how we originally handled PointerEvents.  I did take another look at things and made some changes to our PointerInfo classes and this has now been changed to just:
```
pi.event as IWheelEvent
````